crates/swc_plugin_macro/src/lib.rs

```diff
@@ -25,6 +25,8 @@ fn handle_func(func: ItemFn) -> TokenStream {
     let ret = quote! {
         #func
 
+        use swc_plugin::Serialized;
```

Comment:
this is minor fix to import module in macro correctly to avoid actual plugin have to manually import this.